modules/ROOT/pages/index.adoc

```diff
@@ -18,14 +18,14 @@ security, and scalability.
 
 Anypoint Studio and Flow Designer support Mule application development.
 
-== Domains
+== Mule Domains
 
 Domains enable you to share global configurations that Mule applications need to
 reuse, such as default error handlers, shared properties, scheduler pools, and
-component configurations.
+component configurations. You can only deploy domains when running Mule runtime engine on-premises.
 
-Anypoint Studio supports Mule domain development.
-See xref:shared-resources.adoc[Shared Resources].
+See xref:shared-resources.adoc[Shared Resources] for Mule domain development and configuration using Anypoint Studio. +
```

Comment:
Perhaps "See 'Shared Resources' for information about developing and configuring Mule domains by using Anypoint Studio."?